src/libraries/Native/Unix/System.Globalization.Native/CMakeLists.txt

```diff
@@ -57,24 +57,32 @@ set(NATIVEGLOBALIZATION_SOURCES
     pal_icushim.c
 )
 
+if (NOT EXCLUDE_VERSION_HEADER_FILE)
+    list(APPEND NATIVEGLOBALIZATION_SOURCES
+        ${VERSION_FILE_PATH}
+    )
+endif()
+
 include_directories(${UTYPES_H})
 
-add_library(System.Globalization.Native
-    SHARED
-    ${NATIVEGLOBALIZATION_SOURCES}
-    ${VERSION_FILE_PATH}
-)
+if (BUILD_GLOBALIZATION_SHARED_LIBRARY)
```

Comment:
Other existing shim CMakefiles use `GEN_SHARED_LIB` for this purpose. Do the same for consistency?

https://github.com/dotnet/runtime/search?utf8=%E2%9C%93&q=GEN_SHARED_LIB&type=

Reply:
Good catch, will update.